_includes/tag_list.html

```diff
@@ -1,4 +1,10 @@
 <div id="sidebar">
+  <div id="advertisement">
+    <a href="https://fastruby.io" target="_blank">
+      <img id="fastruby" src="assets/images/upgrade_rails.png" alt="Upgrade Ruby">
+      <p id="ad_copy"> You Need to Upgrade Rails 5.2. We can help with our productized consulting service. </p>
```

Comment:
@ercohen14 The casing looks a little weird here. Maybe we should just make it "You need to upgrade your app to Rails 5.2."?